content/en/load-testing/Advanced/060-cloud-setup.md

```diff
@@ -55,4 +218,259 @@ The Amazon Elastic Compute Cloud service is another perfect fit for on-demand lo
 
 Note that this tool is not intended to replace the <a href="http://aws.amazon.com/" target="_blank">AWS console</a> or similar tools.
 
-{{< TODO comment="more to come" >}}TBD{{< /TODO >}}
\ No newline at end of file
+### Image Templates for AWS 
+
+Xceptance provides AMIs (Amazon Machine Images) for use with Amazon EC2. These AMIs are pre-packaged systems, which are optimized for load testing and have XLT already installed. Using one of these AMIs may save you the work to create and maintain your own AMIs, but may also impose additional costs. Amazon will charge you for the infrastructure usage. Make sure that your security group permits communication on port 8500. This is the XLT agent port on these machines. A list of current AWS AMI ids can be found next to the release information on <a href="https://github.com/Xceptance/XLT/releases" target="_blank">GitHub</a>.
+
+In addition to that, you can also build your own images using our public <a href="https://github.com/Xceptance/XLT-Packer" target="_blank">XLT-Packer project</a>. It contains a nice readme on how to use it to create images set up for running XLT for the cloud vendor of your choice.
+
+### Setting up XLT's ec2_admin tool
+
+Before you can use the tool, you have to **configure** it appropriately. There is a configuration file for this: `<xlt>/config/ec2_admin.properties`. 
+
+The most important settings are your _AWS credentials_. These are needed to authorize any AWS operation, which is executed on your behalf.
```

Comment:
Can we try to figure out which permissions this account needs to fully function?